bp_top/test/tb/bp_tethered/Makefile.params

```diff
@@ -17,6 +17,10 @@ export COSIM_P        ?= 0
 export PRELOAD_MEM_P  ?= 0
 export NBF_CONFIG_P   ?= 1
 
+export SIM_SYNTH_VIVADO_P ?= 0
+export SIM_SYNTH_DC_P     ?= 0
+export NO_BIND_P          := $(shell echo $(SIM_SYNTH_VIVADO_P) || $(SIM_SYNTH_DC_P))
```

Comment:
Should use ?= so we can override it. Was there a problem with lazy evaluation?

Reply:
No, there was no problem. Changed it to ?=